testkit/src/main/scala/org/bitcoins/testkit/chain/SyncUtil.scala

```diff
@@ -84,7 +84,7 @@ abstract class SyncUtil extends BitcoinSLogger {
           case BlockStamp.BlockHeight(height) =>
             Future.successful(height)
           case BlockStamp.BlockTime(_) =>
-            Future.failed(new RuntimeException(s"Cannot query by block time"))
+            Future.successful(0)
```

Comment:
Why is this changed? Bitcoin core doesn't allow you to query by block time AFAIK?